DIPs/DIP1337.md

```diff
@@ -0,0 +1,85 @@
+# Tail Call Optimization requirement
+
+| Section         | Value                                                           |
+|-----------------|-----------------------------------------------------------------|
+| DIP:            | 1337                                                            |
+| Author:         | Dietrich Daroch (name dot surname on gmail)                     |
+| Implementation: | (links to implementation PR if any)                             |
+| Status:         | -                                                               |
+
+
+
+## Abstract
+
+Add an attribute that forces tail call optimization.
```

Comment:
Why can't something similar to `pragma(inline, true)` be used? The rationale should at least cover alternative ideas.